tests/functional/reset_password.js

```diff
@@ -108,8 +110,7 @@ define([
     name: 'reset password',
 
     beforeEach: function () {
-      // timeout after 90 seconds
-      this.timeout = 90000;
+      this.timeout = new Duration('90s').milliseconds();
```

Comment:
All these cases bellow `this.timeout = new Duration('90s').milliseconds();` can just use the `TIMEOUT`  value from above.